clase6/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <form id="formulario">
+      <label for="cantMiembros"
+        >Ingrese cantidad de miembros en el grupo familiar</label
+      >
+
+      <input type="number" id="cantMiembros" />
+
+      <button type="button" id="botonMiembros">Siguiente</button>
+      <button>
+        <a href="" style="text-decoration: none; color: black">Resetear</a>
+      </button>
+
+      <br />
+    </form>
+
+    <button id="botonCalcular">Calcular</button>
+
+    <br />
+    <em id="edadMayor"></em><br />
+    <em id="edadMenor"></em><br />
+    <em id="edadPromedio"></em><br />
+    <br />
+    <em id="salarioMayor"></em><br />
+    <em id="salarioMenor"></em><br />
+    <em id="salarioPromedio"></em><br />
```

Comment:
Los tags son importantes para el SEO y los search engines como Google, y deberían usarse como tal. El <em> es para enfatizar cosas, pero para SEO, no para darle estilo. En este caso, deberías hacerlo con CSS.

https://developer.mozilla.org/es/docs/Web/HTML/Element/em
https://www.socialmediatoday.com/news/8-of-the-most-important-html-tags-for-seo/574987/